### PR TITLE
unbreak, and update deprecated dependencies and practices

### PR DESCRIPTION
**Pull request by Habbie**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,18 +19,18 @@ env:
 jobs:
   matrix-setup:
     name: Matrix Setup
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (dwfreed) [inline .github/workflows/ci.yml:22]: Do you happen to have a log of the failure?

Comment (Habbie) [reply .github/workflows/ci.yml:22]: `Build dependency: Please install Python 2.x`

https://github.com/Habbie/openwrt-vlmcsd/actions/runs/5243824268/jobs/9469004588

Comment (Habbie) [reply .github/workflows/ci.yml:22]: Can probably fix it by installing Python 2.x, but I don't really want to do that, and I did not dig any further yet.

Comment (Habbie) [reply .github/workflows/ci.yml]: addressed by refreshing `matrix.json`, which got rid of 19.07

**Comment by Habbie** [comment]
force pushed to remove the ubuntu 20.04 downgrade

**Review by dwfreed — APPROVED** [review]

**Pull request merged**